src/dali_model_instance.cc

```diff
@@ -109,19 +109,33 @@ std::vector<TritonResponse> DaliModelInstance::CreateResponses(
   return responses;
 }
 
+
 ProcessingMeta DaliModelInstance::ProcessRequests(const std::vector<TritonRequest>& requests,
                                                   const std::vector<TritonResponse>& responses) {
   ProcessingMeta ret{};
+
+  TimeRange tr_gi("[DALI BE] GenerateInputs", 0x488FB1);
   auto inputs_info = GenerateInputs(requests);
+  tr_gi.stop();
+
+  TimeRange tr_run("[DALI BE] Run processing", 0x488FB1);
```

Comment:
You can use predefined constants from TimeRange, not magic values.

Reply:
Done